src/notation/internal/masternotationmididata.cpp

```diff
@@ -157,22 +160,29 @@ std::vector<Event> MasterNotationMidiData::retrieveSetupEvents(const std::list<I
     return result;
 }
 
-Events MasterNotationMidiData::eventsFromRange(const channel_t midiChannel, const tick_t fromTick, const tick_t toTick) const
+Events MasterNotationMidiData::eventsFromRange(const std::vector<channel_t>& midiChannels, const tick_t fromTick, const tick_t toTick) const
```

Comment:
You use std::vector<channel_t> in many places. Wouldn't it be better to create an alias?
MidiChannels = std::vector< channel_t>

Reply:
Yeah, might make sense
I'd fix it in another PR since it's used quite often